Documentation/contributor-guide/bump_etcd_version_k8s.md

```diff
@@ -0,0 +1,167 @@
+# Bump etcd Version in Kubernetes
+
+This guide will walk through the update of etcd in Kubernetes to a new version (`kubernetes/kubernetes` repository). 
+
+You can use this [issue](https://github.com/kubernetes/kubernetes/issues/131101) as a reference when updating the etcd version in Kubernetes.
+
+Bumping the etcd version in Kubernetes consists of two steps. 
+* Bump etcd client SDK
+* Bump etcd image
+
+> The commented lines in this document signifies the line to be changed
+
+## Bump etcd client SDK
+
+> Reference: [link](https://github.com/kubernetes/kubernetes/pull/131103)
```

Comment:
It's good to link to an example PR. But this PR might not be a good example, because it contains too many unrelated changes/commits. Since we don't have better example, so let's just leave it as it's for now.